packages/table/src/ExtendedView.js

```diff
@@ -39,45 +39,39 @@ const ExtendedView = ({
   return (
     <ThemeProvider theme={themeConfig}>
       {(numSelected === 0 && (download || manageViewColumns)) && (
-      <Toolbar className="downloadColumnView">
-        <DownloadButton
-          count={table.totalRowCount || 0}
-          queryVariables={queryVariables}
-          table={table}
-          buttonConfig={downloadButtonConfig}
-        />
-        <ManageColumnView
-          table={table}
-          manageViewColumns={manageViewColumns}
-          onColumnViewChange={onColumnViewChange}
-        />
-      </Toolbar>
+        <Toolbar className="downloadColumnView">
+
+          <ManageColumnView
+            table={table}
+            manageViewColumns={manageViewColumns}
+            onColumnViewChange={onColumnViewChange}
+          />
+        </Toolbar>
       )}
       {
-          (pagination) && (
-            <div className="downloadArea" style={(table.paginationCustomStyle && table.paginationCustomStyle.topPagination) ? table.paginationCustomStyle.topPagination : downloadAreaStyle}>
-              <CustomPagination
-                customTheme={customTheme.tblTopPgn}
-                rowsPerPageOptions={[10, 25, 50, 100]}
-                component="div"
-                count={table.totalRowCount || 0}
-                rowsPerPage={table.rowsPerPage || 10}
-                page={table.page || 0}
-                onPageChange={onPageChange}
-                onRowsPerPageChange={onRowsPerPageChange}
-                queryVariables={queryVariables}
-                table={table}
-              />
-              {/* TODO: Commented out this section, Make this options configrable on Bento. */}
-              {/* <DownloadButton
-                count={table.totalRowCount || 0}
-                queryVariables={queryVariables}
-                table={table}
-                isIcon={false}
-              /> */}
-            </div>
-          )
-        }
+        (pagination) && (
+          <div className="downloadArea" style={(table.paginationCustomStyle && table.paginationCustomStyle.topPagination) ? table.paginationCustomStyle.topPagination : downloadAreaStyle}>
+            <CustomPagination
+              customTheme={customTheme.tblTopPgn}
+              rowsPerPageOptions={[10, 25, 50, 100]}
+              component="div"
+              count={table.totalRowCount || 0}
+              rowsPerPage={table.rowsPerPage || 10}
+              page={table.page || 0}
+              onPageChange={onPageChange}
+              onRowsPerPageChange={onRowsPerPageChange}
+              queryVariables={queryVariables}
+              table={table}
+            />
+            <DownloadButton
```

Comment:
@Senait7-ux Is this configureable?

Reply:
If by configureable we mean, "is the download option visible or not based on a configurable variable?" no, but maybe if you can give me some specific things that should be configurable I can answer the question more clearly